test/functional/deprecated_rpc.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test deprecation of RPC calls."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
```

Comment:
nit: prefer explicit imports over wildcard imports. Here, you're just using `assert_raises_jsonrpc` from util.py, so you can call:

```python
from test_framework.util import assert_raises_jsonrpc
```